NuRadioReco/modules/channelResampler.py

```diff
@@ -39,12 +23,15 @@ def run(self, evt, station, det, sampling_rate):
 
         evt, station, det
             Event, Station, Detector
-        sampling_rate: float
+        sampling_rate: float (Default: None)
             In units 1/time provides the desired sampling rate of the data.
```

Comment:
This is a neat feature! Can you add what happens if sampling_rate=None to the docstrings?

Reply:
fine?